nss-day-cohort-41/tabloid-fullstack-bat-boy-apologists
# post image

# Description
Allows user to upload an image from their computer instead of pasting a url. 
Edit post functionality not working at the moment.

Fixes # (issue)

## Type of change

# Testing Instructions
While inside the client folder, `npm install axios` to install axios

Restart Visual Studio
Make a new post and upload image
Image should be saved in ~client/public folder
Image should show when you click on post details

# Checklist:
